charts/ambassador/templates/rbac.yaml

```diff
@@ -1,14 +1,66 @@
 {{- if .Values.rbac.create -}}
+---
```

Comment:
so... i apologize for putting this change in here, but the reason for it will be more clear when generating the manifests for the apro.git e2e tests.

Basically, the really hard/annoying part of that was ambassador single namespace. The cluster wide CRD role was always needed, and if ambassador is single namespace, it creates another Role for the namespaced resources. if ambassador is not single namespaced, it would just append the rules to the cluster role with the CRDs. Separating our the roles in this way made generating the flat yaml for the single namespace situation much easier, and i just generally think this makes it easier to manage.